tailwind.config.js

```diff
@@ -1,8 +1,71 @@
+const colors = require('tailwindcss/colors');
+
 module.exports = {
   purge: ['./src/**/*.{js,ts,jsx,tsx}'],
-  darkMode: false, // or 'media' or 'class'
+  darkMode: false,
   theme: {
-    extend: {},
+    fontFamily: {
```

Comment:
I know that this is probably copy-pasted from `@smartive/guetzli` but in your case I would _extend_ the theme and not override it. This way you lose all colors from tailwind.